### PR TITLE
Create a configuration for tf/minigo

### PR DESCRIPTION
**Pull request by Kashomon**
This creates a prow configuration for tensorflow/minigo. I'm quite the noob at this -- most of this is copy pasta, so let know what magic strings need to be changed =).

### CODE REVIEW

**Comment by Kashomon** [comment]
@cjwagner -- since we were talking about this earlier.

**Comment by Kashomon** [comment]
Also, ping @amj for FYI.

File: prow/config.yaml [hunk 1]
```diff
@@ -5376,6 +5376,20 @@ presubmits:
...
 
+  tensorflow/minigo:
+  - name: tf-minigo-presubmit
+    context: tf-minigo-presubmit
+    agent: kubernetes
+    always_run: true         # Run for every PR, or only when requested.
+    branches:
+    - master
+    labels:
+      preset-service-account: true
```
Comment (krzyzacy) [inline prow/config.yaml]: it might not need our service account? Where's the job upload logs to?

Comment (BenTheElder) [reply prow/config.yaml]: Not sure if this is intended but this also doesn't seem to be doing git checkout etc. ?

Comment (Kashomon) [reply prow/config.yaml]: It doesn't upload the logs anywhere. But it seems like that would be a useful feature! What's the pattern here?

**Comment by Kashomon** [comment]
By the way, there's a corresponding PR in tensorflow/minigo to make the test harness: https://github.com/tensorflow/minigo/pull/40

File: prow/config.yaml (hunk 1 above)
Comment (cjwagner) [reply prow/config.yaml]: I left a comment related to this [here](https://github.com/tensorflow/minigo/pull/40#discussion_r165541198).

**Comment by Kashomon** [comment]
I'm still getting

Testgrid group tf-minigo-periodic does not have a matching jenkins or prow job
Testgrid group tf-minigo-postsubmit does not have a matching jenkins or prow job
Total bad config(s) - 2
--- FAIL: TestJobsTestgridEntryMatch (0.85s)
	config_test.go:460: Failed with invalid config or job entries
FAIL

What does this mean?

**Comment by BenTheElder** [comment]
OTOH it's probably failing because your jobs don't actually set anything to do log uploads?
/assign @krzyzacy

File: prow/config.yaml [hunk 2]
```diff
@@ -5979,6 +5995,18 @@ postsubmits:
       - image: gcr.io/mlkube-testing/builder:latest
         imagePullPolicy: Always
 
+  tensorflow/minigo:
+  - name: tf-minigo-presubmit
```
Comment (krzyzacy) [inline prow/config.yaml]: s/presubmit/postsubmit

File: testgrid/config/config.yaml [hunk 3]
```diff
@@ -1677,6 +1677,13 @@ test_groups:
   num_columns_recent: 30
 - name: tf-k8s-postsubmit
   gcs_prefix: kubernetes-jenkins/logs/tensorflow_k8s/tf-k8s-postsubmit
+- name: tf-minigo-periodic
+  gcs_prefix: kubernetes-jenkins/logs/tf-minigo-periodic
```
Comment (krzyzacy) [inline testgrid/config/config.yaml]: and you didn't add a periodic job in prow

Comment (Kashomon) [reply testgrid/config/config.yaml]: Done

File: prow/config.yaml (hunk 2 above)
Comment (Kashomon) [reply prow/config.yaml]: Done

**Comment by krzyzacy** [comment]
add an entry to https://github.com/kubernetes/test-infra/blob/master/testgrid/cmd/config/config_test.go#L352-L362

I guess we can change the test to use a blacklist instead, but this will make presubmit happier

**Comment by Kashomon** [comment]
Ok, I'm not sure what I'm doing wrong here: 

```
Testgrid group tf-minigo-presubmit does not have a matching jenkins or prow job
Total bad config(s) - 1
```

But it does exist in config.yaml:

```yaml
  tensorflow/minigo:
  - name: tf-minigo-presubmit
    context: tf-minigo-presubmit
    agent: kubernetes
    always_run: true         # Run for every PR, or only when requested.
    rerun_command: "/test tf-minigo-presubmit"
    trigger: "(?m)^/test( all| tf-minigo-presubmit),?(\\s+|$)"
    branches:
    - master
    labels:
      preset-service-account: true
    spec:
      containers:
      - image: gcr.io/minigo-testing/minigo-prow-harness:latest
        imagePullPolicy: Always
```

**Comment by krzyzacy** [comment]
see my previous comment about the testgrid config test

and have you configured webhook from your repo to send to prow yet?

**Pull request merged**